WooCommerce/WooCommerceTests/Tools/DefaultImageServiceTests.swift

```diff
@@ -147,4 +149,121 @@ final class DefaultImageServiceTests: XCTestCase {
         waitForExpectations(timeout: Constants.expectationTimeout, handler: nil)
     }
 
+    func testRetrieveImageWithCachingFlow() {
```

Comment:
Please follow the team's pattern for test function naming for better readability. Also, the naming should clarify the condition and expectation.

```suggestion
    func test_retrieveImage_caches_image_on_second_retrieval_when_caching_is_enabled() {
```

Reply:
The same comment applies to the tests below.